src/components/SigningModals/TransactionJourney.js

```diff
@@ -0,0 +1,265 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { css, keyframes } from 'styled-components'
+import { textStyle, useTheme, GU, RADIUS } from '@aragon/ui'
+
+const PREVIOUS = 'PREVIOUS'
+const CURRENT = 'CURRENT'
+const NEXT = 'NEXT'
+
+const ITEM_SPACING = 3.5 * GU
+
+const pulseAnimation = keyframes`
+  from {
+    transform: scale3d(0.25, 0.25, 1);
+    opacity: 0.5;
+  }
+
+  to {
+    transform: scale3d(1.25, 1.25, 1);
+    opacity: 0;
+  }
+`
+
+function getItemStatus(i, activeItem) {
+  if (i < activeItem) {
+    return PREVIOUS
+  }
+
+  if (i > activeItem) {
+    return NEXT
+  }
+
+  return CURRENT
+}
+
+function TransactionJourney({ items, activeItem, pulse, themeColor }) {
+  const theme = useTheme()
+  const color = themeColor || theme.accent
+
+  const renderItems = () =>
```

Comment:
Let's inline! In this case it's pretty simple and I think easier to follow with just one render location :)